### DISCUSSION TITLE
Towards an even larger SYNERGY dataset

### DISCUSSION THREAD

**Discussion by J535D165** [asreview/synergy-dataset#97] [General]
SYNERGY is a free and open dataset on study selection in systematic reviews, comprising 169,288 academic works from 26 systematic reviews. Although this is already a large dataset compared with other resources, we aim to develop an even larger dataset: SYNERGY+ (SYNERGY Plus). This dataset contains even more records, datasets, a test-train split, and many more improvements. For the development of this dataset, we would appreciate your help. In the upcoming weeks, we will update this thread with information and instructions on how to contribute to this dataset. 

Subscribe to this thread to keep up-to-date 🙏

**Comment by sidgoogle** [comment]
Hi, just curious if there is an update. Thanks for the valuable resource.

**Reply by EmilyWes** [reply]
Hi sidgoogle, 

We are long due for an update, so thank you for the question :).

- We have started actively gathering more datasets, both by searching online and reaching out to people.
- We are standardizing and improving the current pipeline for making those datasets available. A number of changes are already in (see the development/v2 branch). When this is done we will start adding datasets, of which a number are already finalized.
- We are also busy refining and extracting information about the datasets, which we plan add over time. An example is the addition of the eligibility criteria, that can be found [here](https://github.com/asreview/synergy-dataset/pull/115). Other things are labels for abstract screening and indicators of various quality criteria of the datasets. 

So keep an eye on mainly the development/v2 branch, where most of this will be added first. And feel free to come with any suggestions / possible improvements :).

**Comment by SamyAteia** [comment]
Are the studies from the CLEF TAR challenge included in the SYNERGY dataset, or are there plans to include them?
https://github.com/CLEF-TAR/tar

**Reply by EmilyWes** [reply]
Hi SamyAteia,

Thank you for the suggestion! 
I have actually looked in to the CLEF dataset. Reading back my notes I understood that the creators of the dataset reproduced the search(es) rather than having the original list of identifiers or DOI's. Since we found that reproducing a search always gives different results, we did not trust the records of the dataset enough to include it in SYNERGY.
However if I understood incorrectly, please let met know so that I can check it again :).

Also nice to know: we also plan to put an overview of the reviews/data we have looked at here on the github repo, but are still deciding what would be the format for that. Reasoning why something is included/excluded in Synergy will definitely be information provided!

**Comment by Mohammadsaknini** [comment]
I have collected some data in a similar fashion to yours. Feel free to add it to your dataset [Repo](https://github.com/Mohammadsaknini/LitQEval/tree/main/data)

**Reply by EmilyWes** [reply]
Thank you very much! I will look into it and let you know if I have any questions :)

**Comment by puzzledhedgehog** [comment]
Hi folks -- very interesting project!
Q1  I see that the title-abstract screening decision is optional; only the full-text screening decision must be included. Is that wise? In many cases, a record can't be excluded by humans based on the title and abstract; only when the full text is considered can the paper be excluded. I don't see how we can reasonably expect a ML tool that doesn't have access to the full text to make accurate full-text screening decisions. (OK, I will admit that in simulations, I was able to get higher performance from ASReview when I tried to achieve 95% recall of the included-after-full-text-screening records, rather than 95% recall of the not-yet-excluded-after-title-abstract-screening records. But the edge cases still worry me.)
Q2 Would you be interested in knowing whether a given document that was screened in or out was identified via database searching or via other methods (citation chaining, hand-searching, etc)?

**Reply by EmilyWes** [reply]
Hi there,

Q1: 
I split the question in to 2 parts:
- Why are full-text labels required and title-abstract (ti-ab) labels not?
Ideally we want both the full-text and the title-abstract labels, but as the main goal for a systematic review is to find the full text inclusions, we need those labels to say something about how well ASReview performs. The title-abstract labels enable a lot more ways to simulate and test strategies, so when published, we include these labels as well.
Note that in general the title-abstract labels are a lot more noisy, making the full-text labels a better candidate to use.

-  "I don't see how we can reasonably expect a ML tool that doesn't have access to the full text to make accurate full-text screening decisions."
ASReview is a ranking tool. It helps the screener to look at the most promising records first, but it won't make a decision about a record itself. So the task is to find the records that are most likely a full-text inclusion, for which we have seen in a lot of cases it can do very well.

Q2: 
That's a very good question. For the new improved version of SYNERGY we are looking at these type of things. Currently, that means registering how many records are found via these methods, and how many are in the actual data published online.
For most datasets, it is the case that the records published online are those from the database searching only.

**Reply by J535D165** [reply]
I would like to add to this the following: 

Synergy contains an extensive set of metadata fields, the title and abstract of which are the most well-known and used. You can find more information on the available metadata fields in the project's README. These metadata fields can be used to derive rich features for classification. For example, authors, keywords, publication year, and concepts are available. You can even access the open-access PDF or link to the full text. Synergy is just the start of rethinking how we conduct systematic reviews or sparse data retrieval for academic literature. We would love to hear about your creative ideas.

**Reply by JJBernieD** [reply]
In our work, we use the letters _**S**_ to denote the starting data set, which is typically the entire set of results from some database search. Selection proceeds in successive rounds, the first one of which is typically based on Title and Abstract, to give the set _**S'**_, followed by _**S''**_, _**S'''**_, ..., which are selected based on full text and sometimes other information, all the way to set _**C**_ (for Cited), the final set of the review. 

The 26 Synergy datasets give _**S**_ and _**C**_. Which ones also have inclusion data for both _**S'**_ and _**C**_?

The first round of selection is where machine-assisted ranking is used.
We too wonder what we learn from using _**C**_ to define the "included" set in simulations that do not use full text. For example, if the model can learn the important features of _**C**_ as efficiently, or more so, than those of _**S'**_, should we conclude that the machine is able to pick up features of _**C**_ from the title and abstract that are visible to the human expert only when they access the full text?
 
That question is meant facetiously; for the simulation, we provide the machine with  _**C**_. It does not select it on its own. The machine would model any arbitrary set of "includeds" we would provide. However, maybe it suggests that if the human experts decided to consult the full text _in the first round_ of the screening process, i.e. the round where software like ASReview can help, then the gain in efficiency would be even greater than when they use only title and abstract.

**Reply by EmilyWes** [reply]
Hi!

To answer your first question:
From the first SYNERGY release (26 datasets), the following also uploaded labels corresponding to set S': Appenzeller-Herzog_2019, Chou_2003, Moran_2020, Nelson_2002, van_de_Schoot_2018, van_der_Valk_2021, van_Dis_2019
- Note 1: We did not extract these labels for the first SYNERGY release, so they are not available there.
- Note 2: For SYNERGY Plus we are extracting S' and make them available, currently we have these labels for 49/89 datasets. (We plan/hope to release SYNERGY Plus this year)

I really like your further questions, as I have asked them myself as well :). Hopefully with SYNERGY Plus, more definitive answers can be found, it is definitively one of the reasons I wanted to include the ti/ab labels (title/abstract being my name for S') wherever they were available.
Although not scientifically researched yet (to my knowledge), I did see several projects that indicated that consulting the full text during the first round of screening is indeed beneficial, at least in combination with ASReview. So I look forward to seeing more extensive research in this direction.

**Reply by JJBernieD** [reply]
These questions are of high interest to us and we look forward to the next release. In the meantime, how do I get the ti/ab labels column outputted to .csv versions of the datasets for which that column is populated? 
Thanks to Jonathan, I have been using `synergy-dataset get --vars doi,title,abstract,publication_year` to output year. What do I add to get the ti/ab label?

In the field, consulting full text during the machine-assisted round of screening, where the "machine" only accesses ti/ab, is a trade off between the efficiency gain that comes from faster and better training of the model and more accurate decisions, and the additional time required to open and read the full text.
I advise screeners to go to full text in that first round only in cases where the information they would need to make the decision is likely to be found very quickly. Say, did the authors report a specific outcome of interest? Did they include the species of interest in the study?

Our limited tests show that even when the deciding information the human expert uses to make the decision was not visible to them in ti/ab, the speed at which full recall is reached is many times greater when the simulation uses what we call _**C**_ rather than _**S'**_.
Does that mean that the machine extracts and models meaningful information that is present in ti/ab but not visible to the human? That would be interesting.
Maybe, but it could also just be that _**C**_ is typically much smaller than _**S'**_, and thus can be pushed up the ranks and labeled faster, because there are just so much fewer records to include. That would be trivial. 
We need to find a way to distinguish the two experimentally.

**Reply by EmilyWes** [reply]
You cannot get the ti/ab labels using any command in the command line, since the labels are not in the first release.

To get the ti/ab labels before the next release requires manual/technical work, since they were not extracted yet by us.
I would see 2 options:
- Use `synergy_dataset show [dataset_name]` to get the location of the original data, and extract the labels from there yourself.
- Get the labels from our development branch of [our Github page](https://github.com/asreview/synergy-dataset/tree/development/v2/datasets).

Both options require a bit of know how to match the records with the ones in the published dataset. The easiest method is to wait for the next release, because there we will make sure you can simple get them via the command line.

**Comment by JJBernieD** [comment]
Oh, alright. I thought you meant that the ti/ab labels were already in Release 1 of Appenzeller-Herzog_2019, Chou_2003, Moran_2020, Nelson_2002, van_de_Schoot_2018, van_der_Valk_2021, and van_Dis_2019. 
Looking forward to Release 2. 

Any predicted date?

**Reply by EmilyWes** [reply]
Ah, yeah, I see how that may have been unclear.
We hope to release this year, but don't have a specific date in mind.

**Comment by JJBernieD** [comment]
Have any datasets with complete label_abstract_included data been finished and do you have any news on release date?

Watching this development with great interest!

**Reply by EmilyWes** [reply]
Hi!

We have finished all the extra cleaning steps that we wanted to take, making sure that the abstracts in all datasets are of high quality. 
This means we now (likely) have a final version of the data.
We are running some last tests and finishing up the paper we are writing about the process, and will then put everything online together.
Assuming no further hinderances my expectation would be within a month.

**Reply by JJBernieD** [reply]
This is fantastic news! We will be on the lookout.

Thank you!!

**Comment by JJBernieD** [comment]
I saw the new release is out! This is a enormous achievement, congratulations!

I have found https://github.com/asreview/synergy-dataset-py.

Congratulations again!

Jean-Jacques

**Reply by EmilyWes** [reply]
Thank you :). 

Indeed the synergy-dataset-py has been updated to be used with both the new (default) and old version of the dataset.

Note that we are in the process of publishing a paper about the creation of the dataset, for which a preprint will be online shortly. When the preprint is available, we will update some links to it and make some more noise that the dataset is online :)

Emily
